charts/kueue/templates/rbac/batch_admin_role.yaml

```diff
@@ -2,6 +2,8 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
```

Comment:
Could you explain why this is needed (and for other RBAC files)?

Reply:
It's not strictly needed, though this can be convenient to have these labels added, for example, for clean-up, so it's possible to perform a delete collection with the appropriate label selector.

Also, it makes the Helm charts consistent with Kustomize, that applies the common labels to all resources, including RBAC ones.

Reply:
Sgtm, thanks for explaining